src/NServiceBus.Core.Tests/Serializers/SerializeMessagesBehaviorTests.cs

```diff
@@ -8,15 +8,20 @@
     using NServiceBus.Unicast;
     using NServiceBus.Unicast.Messages;
     using NUnit.Framework;
+    using Conventions = NServiceBus.Conventions;
 
     public class SerializeMessagesBehaviorTests
     {
         [Test]
         public void Should_set_content_type_header()
         {
-            var behavior = new SerializeMessagesBehavior(new FakeSerializer("myContentType"));
-          
-            var context = new OutgoingContext(null, new SendMessageOptions("test"), new LogicalMessage(new MessageMetadata(),null,null),new Dictionary<string, string>(),"msg id",MessageIntentEnum.Send);
+            var registry = new MessageMetadataRegistry(true,new Conventions());
```

Comment:
Totally nitpicker I know but can we agree on the space after the comma or not in general?